sweepai/handlers/on_ticket.py

```diff
@@ -59,7 +59,7 @@
 sep = "\n---\n"
 bot_suffix_starring = "⭐ If you are enjoying Sweep, please [star our repo](https://github.com/sweepai/sweep) so more people can hear about us!"
 bot_suffix = (
-    f"\n{sep} To recreate the pull request edit the issue title or description."
+    f"\n{sep} 💡 To recreate the pull request edit the issue title or description."
```

Comment:
add another line between sep and the lightbulb with the latest changes, injecting "Latest changes!\n* You can now get Sweep to format files before pushing by adding content from https://docs.sweep.dev/config#sandbox to you Sweep.yaml!" followed by another seperator. Make sure to add an appropriate amount of newlines so everything renders properly.